### ISSUE TITLE
AI depth 8 crashes

### ISSUE THREAD

**Issue by potatoeggy** [potatoeggy/connectfour#38]
```
Exception in thread "main" java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at java.util.ArrayList.rangeCheck(ArrayList.java:657)
        at java.util.ArrayList.get(ArrayList.java:433)
        at AI.bestColumn(AI.java:219)
        at MainWindow.main(MainWindow.java:154)
```

when facing two "impossible" ais

**Comment by vincentjguo** [comment]
uh yes don't make the AI face itself. Can you make it so when one player chooses computer the other dropdown box is disabled?

**Comment by potatoeggy** [comment]
Apologies, this also happens when computers play against humans. AI vs AI works fine on all other difficulty levels.

**Comment by vincentjguo** [comment]
Hmm, can you send me the code you used to make the AI face itself?

**Comment by potatoeggy** [comment]
I set both combo boxes to "Computer".

**Comment by potatoeggy** [comment]
in the timer branch, although this might happen in ai_gui.

**Comment by potatoeggy** [comment]
It's actually completely broken now.

**Comment by potatoeggy** [comment]
Regression has been fixed

**Comment by potatoeggy** [comment]
But now that the alpha beta stuff has been removed, turns take absolutely forever in impossible mode, so I cannot tell if it crashes or not.

**Comment by potatoeggy** [comment]
Fixed in 661f221c7042fe30b738282bf45d7855ab7dba75
